src/prefect/main.py

```diff
@@ -67,6 +67,9 @@
     "flow",
     "Flow",
     "get_client",
+    "PrefectClient",
+    "get_cloud_client",
+    "CloudClient",
```

Comment:
I don't think we should expose `PrefectClient` and `CloudClient` at the top level and prefer `get_client` and `get_cloud_client` instead.
```suggestion
    "get_cloud_client",
```

Reply:
@desertaxle that makes sense, do you happen to know why they're imported in `prefect.main`? i.e. should I remove them from `prefect.main` entirely?

Reply:
I'm not sure why they're imported here, but based on the fact that they weren't included in `__all__`, I think we can remove them.

Reply:
cool, removed in [883ec70](https://github.com/PrefectHQ/prefect/pull/16288/commits/883ec7029822be4c4f37515e6903fdd01b6ad75b)